pkg/kubelet/eviction/api/types.go

```diff
@@ -40,6 +40,13 @@ const (
 	SignalAllocatableMemoryAvailable Signal = "allocatableMemory.available"
 	// SignalPIDAvailable is amount of PID available for pod allocation
 	SignalPIDAvailable Signal = "pid.available"
+
+	// SignalEphemeralContainerFsLimit is amount of storage available on filesystem requested by the container
```

Comment:
Just double checking: Do we need to add these new signals to the `OpForSignal` dict?